services/cinema_hall.py

```diff
@@ -0,0 +1,16 @@
+from django.db.models.query import QuerySet
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(hall_name: str,
+                       hall_rows: int,
+                       hall_seats_in_row: int
+                       ) -> None:
+    CinemaHall.objects.create(name=hall_name,
+                              rows=hall_rows,
+                              seats_in_row=hall_seats_in_row
+                              )

```

Comment:
Please provide the same style as I suggested you above

Reply:
fixed

Reply:
fixed